std/algorithm/iteration.d

```diff
@@ -1319,7 +1319,7 @@ private struct FilterBidiResult(alias pred, Range)
     }
 }
 
-// group
+/// group
 struct Group(alias pred, R) if (isInputRange!R)
```

Comment:
This should be under the same docs as its factory function. Same goes for all of the public range structs.